api/src/main/java/org/apache/cloudstack/api/command/user/iso/RegisterIsoCmd.java

```diff
@@ -113,6 +113,11 @@
             description = "true if ISO should bypass Secondary Storage and be downloaded to Primary Storage on deployment")
     private Boolean directDownload;
 
+    @Parameter(name = ApiConstants.PASSWORD_ENABLED,
+            type = CommandType.BOOLEAN,
+            description = "true if password reset feature is supported; default is false")
+    private Boolean passwordEnabled;
```

Comment:
@anuragaw let's export this via the template response (listIsos response should return this key) and we're golden!

Reply:
Added this in the 3rd commit @rhtyd . This was also prohibiting adding tests for update iso cmd in test_iso which works well now. Thanks!